src/main/java/gregtech/common/metatileentities/multi/multiblockpart/MetaTileEntityPassthroughHatchItem.java

```diff
@@ -80,7 +80,7 @@ protected IItemHandlerModifiable createExportItemHandler() {
     protected IItemHandlerModifiable createImportItemHandler() {
         return new NotifiableItemStackHandler(getInventorySize(), getController(), false);
     }
-
+/*
```

Comment:
Why are these commented out? Seems like irrelevant changes for the PR, plus something we don't really want to change.

Reply:
They were commented out because of bad imports preventing me from using `runClient`, but the imports have since been fixed so i'll uncomment these sections.